python-sdk/src/astro/databases/snowflake.py

```diff
@@ -648,8 +648,10 @@ def load_pandas_dataframe_to_table(
         self._assert_not_empty_df(source_dataframe)
 
         auto_create_table = False
-        if if_exists == "replace" or not self.table_exists(target_table):
+        if not self.table_exists(target_table):
             auto_create_table = True
+        elif if_exists == "replace":
```

Comment:
What would happen when the table doesn't exist and the user has passed in `replace`?

Reply:
`auto_create_table` would be true and pandas will take care of creating table